src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -32,60 +32,118 @@ public GivenThatWeWantToPublishAnAotApp(ITestOutputHelper log) : base(log)
 
         [RequiresMSBuildVersionTheory("17.0.0.32901")]
         [InlineData(LatestTfm)]
-        public void ILLink_aot_analyzer_warnings_are_produced(string targetFramework)
+        public void Trimming_warnings_not_produced_if_only_EnableAotAnalyzer_is_set(string targetFramework)
```

Comment:
It will be great to have more comments on this - it looks like aot warnings are generated and it might be a little confusing for folks who don't know the distinction between aot and trim warnings ranges

Reply:
Let me know if the new comments/test renaming clarifies the goal of the tests :D